60_networks.Rmd

```diff
@@ -0,0 +1,664 @@
+
+```
+
+## Data preparation
+
+The PeerJ data set [@potbhare2022skin] is used in this example, which contains skin microbial profiles of 58 subjects. 
+
+```{r load_data}
+data("peerj13075", package = "mia")
+```
+
+Since NetCoMi expects either a count matrix or a phyloseq object as input, we first transform the data into a phyloseq object. This object contains the count matrix, the taxonomy table, and a list with metadata.
+
+```{r make_phyloseq}
+peerj_phy <- makePhyloseqFromTreeSE(peerj13075)
+peerj_phy
+head(tax_table(peerj_phy))
+```
+Since we want to construct the networks at genus level, we need to make the genus names unique. NetCoMi's function `renameTaxa` does this job for us by numerating duplicate names. 
+
+The function can also be used to rename missing genus names by replacing them with the names of higher taxonomic ranks. For this data set this step is not necessary as there are no missing genus names.
+
+```{r rename_taxa}
+# Check the genus names for duplicates
+any(duplicated(tax_table(peerj_phy)[, "genus"]))
+
+# Rename taxonomic table and make rank genus unique
+peerj_phy <- renameTaxa(peerj_phy, 
+                        pat = "<name>",
+                        numDupli = "genus",
+                        ranks = colnames(tax_table(peerj_phy)))
+
+any(duplicated(tax_table(peerj_phy)[, "genus"]))
+```
+## Analyzing a single network
+
+### Network construction
+
+We construct a single association network using SPRING [@yoon2019microbial] to estimate partial correlations between genera. In contrast to classical correlation measures such as Pearson's correlation coefficient, partial correlations can be interpreted as direct associations between microbes. Under certain conditions, two microbes with a partial correlation of zero can be considered conditionally independent.
+
+The data are filtered within `netConstruct` as follows:
+
+* Only samples with a total of at least 1000 reads are included (argument `filtSamp`).
+* Only the 50 most abundant taxa are included (argument `filtTax`).
+
+The argument `measure` defines the association or dissimilarity measure, in our case the SPRING approach. Additional arguments are passed to `SPRING` via `measurePar`. `nlambda` and `rep.num` are the length of the $\lambda$ vector and the repetition number of subsampling used for model selection.The "approx" method (multilinear approximation of latent correlations) is used to save runtime.
+
+Normalization and zero handling is done internally in `SPRING`, so we set `normMethod` and `zeroMethod` to "none".
+
+We also set `sparsMethod` to "none" because the SPRING approach includes model selection to obtain a sparse network. Therefore, no additional sparsification step is needed.
+
+The "signed" method is used to transform associations into dissimilarities (argument `dissFunc`). Thus, strongly negatively associated taxa have a high dissimilarity and conversely a low similarity, corresponding to low edge weights on the network plot.
+
```

Comment:
Consider carefully how much of these technical details to provide here vs. the package documentation. 

The general philosophy:
1. OMA provides general guidance on implementing useful analyses and often this focuses on the application problem & general concepts more than individual package. OMA links to the individual packages for more information.
2. The R packages own website/vignette/manpages will contain more technical details and examples that complement and expand material in OMA.

Here, I would suggest moving some of the details to package's own tutorial and linking from here. There are no exact guidelines, let us consider it from the user perspective and view OMA as a general, approachable introduction to microbiome data science methodology.